packages/medusa/src/migrations/1679921553298-product_category_indexes.ts

```diff
@@ -0,0 +1,33 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
```

Comment:
question: It feels like we are indexing all the columns of the entity. One thing is that doing such thing can end up being the same as not indexing as all the data will end up in the indexes. any particular reason why we don't create an index for example on all non internal and active categories?